apps/editing-toolkit/webpack.config.js

```diff
@@ -32,40 +86,25 @@ const isDevelopment = process.env.NODE_ENV !== 'production';
 function getWebpackConfig( env = { source: '' }, argv = {} ) {
 	env.WP = true;
 
-	// object provides ability to name the entry point
-	// which enables dynamic file names
-	const sources = env.source.split( ',' );
-
-	// Output path
-	let packageName;
-	const entry = sources.reduce( ( accumulator, source ) => {
-		const sourceSegments = source.split( path.sep );
-		const scriptName =
-			sourceSegments.length > 1 ? sourceSegments.slice( 1 ).join( path.sep ) : source;
-
-		// All outputs from a particular package must share an output path.
-		const sourcePackage = sourceSegments[ 0 ];
-		if ( packageName && packageName !== sourceSegments[ 0 ] ) {
-			throw new Error( 'FSE can build multiple sources for a single package.' );
-		}
-		packageName = sourcePackage;
-
-		accumulator[ scriptName ] = path.join( __dirname, 'editing-toolkit-plugin', source );
-		return accumulator;
-	}, {} );
-
-	const outputPath = path.join( __dirname, 'editing-toolkit-plugin', packageName, 'dist' );
-
 	const webpackConfig = getBaseWebpackConfig( env, argv );
 
-	return {
+	const config = {
 		...webpackConfig,
-		entry,
 		output: {
-			...webpackConfig.output,
-			path: outputPath,
-			filename: '[name].js', // dynamic filename
 			library: 'EditingToolkit',
+			path: path.join( __dirname, 'editing-toolkit-plugin' ),
+		},
+		entry: generateEntry( ETK_ENTRYPOINTS ),
+		module: {
+			...webpackConfig.module,
+			rules: [
+				// Remove the default asset loader.
+				...webpackConfig.module.rules.filter( ( rule ) => rule.type !== 'asset/resource' ),
+				// Add our own asset loader, placing images in a more appropriate location.
+				FileConfig.loader( {
+					outputPath: 'dist/images',
```

Comment:
Otherwise, files just go in a folder called "images" at the top level, which is harder to ignore.